pywinauto/win32functions.py

```diff
@@ -180,14 +196,50 @@
 except AttributeError:
     SetWindowLongPtr = SetWindowLong
 SystemParametersInfo =	ctypes.windll.user32.SystemParametersInfoW
-VirtualAllocEx		=	ctypes.windll.kernel32.VirtualAllocEx
-VirtualAllocEx.restype = ctypes.c_void_p
-VirtualFreeEx		=	ctypes.windll.kernel32.VirtualFreeEx
+VirtualAllocEx = ctypes.windll.kernel32.VirtualAllocEx
+VirtualAllocEx.restype = wintypes.LPVOID
+VirtualAllocEx.argtypes = [
+    wintypes.HANDLE,
+    wintypes.LPVOID,
+    ctypes.c_size_t,
+    wintypes.DWORD,
+    wintypes.DWORD
+]
+VirtualFreeEx =	ctypes.windll.kernel32.VirtualFreeEx
+VirtualFreeEx.restype = wintypes.BOOL
+VirtualFreeEx.argtypes = [
+    wintypes.HANDLE,
+    wintypes.LPVOID,
+    ctypes.c_size_t,
+    wintypes.DWORD
+    ]
 DebugBreakProcess	=	ctypes.windll.kernel32.DebugBreakProcess
 
-VirtualAlloc		=	ctypes.windll.kernel32.VirtualAlloc
-VirtualFree			=	ctypes.windll.kernel32.VirtualFree
-WriteProcessMemory	=	ctypes.windll.kernel32.WriteProcessMemory
+VirtualAlloc = ctypes.windll.kernel32.VirtualAlloc
+VirtualAlloc.restype = wintypes.LPVOID
+VirtualAlloc.argtypes = [
+    wintypes.LPVOID,
+    ctypes.c_size_t,
+    wintypes.DWORD,
+    wintypes.DWORD

```

Comment:
(minor) Comma can be after every element to simplify potential copy-paste or `Ctrl+D`.